_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -2409,174 +2476,304 @@
+/turf/open/floor/plating,
+/area/ship/maintenance/port)
+"Xx" = (
+/obj/docking_port/stationary{
+	dir = 8;
+	dwidth = 15;
+	height = 15;
+	width = 30
+	},
+/turf/template_noop,
+/area/template_noop)
+"XL" = (
+/obj/machinery/power/terminal{
 	dir = 8
 	},
-/obj/machinery/door/firedoor/border_only,
-/obj/machinery/door/firedoor/border_only{
-	dir = 1
+/obj/structure/cable/yellow{
+	icon_state = "0-4"
+	},
+/obj/item/wrench/crescent,
+/obj/item/radio/intercom/directional/north,
+/turf/open/floor/plating,
+/area/ship/maintenance/port)
+"Ym" = (
+/obj/machinery/holopad/emergency/command,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 10
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 9
+	},
+/obj/structure/cable{
+	icon_state = "2-8"
+	},
+/turf/open/floor/carpet/cyan,
+/area/ship/bridge)
+"Yn" = (
+/obj/structure/curtain/cloth{
+	color = "#ACD1E9"
+	},
+/obj/structure/grille,
+/obj/structure/window/reinforced/fulltile/shuttle,
+/turf/open/floor/plating,
+/area/ship/bridge)
+"Yw" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
 	},
-/obj/structure/extinguisher_cabinet/directional/west,
-/turf/open/floor/plasteel/white,
-/area/ship/medical)
+/obj/item/storage/toolbox/electrical,
+/obj/machinery/light/small/directional/south,
+/turf/open/floor/plating,
+/area/ship/maintenance/port)
 "YM" = (
 /turf/closed/wall/mineral/titanium/nodiagonal,
 /area/ship/medical)
-"Zb" = (
-/obj/effect/spawner/structure/window/shuttle,
-/obj/machinery/door/poddoor/preopen{
-	dir = 4;
-	id = "traumawindows";
-	name = "Window Blast Door"
+"YQ" = (
+/obj/effect/turf_decal/number/five,
+/obj/machinery/power/apc/auto_name/directional/north,
+/obj/machinery/light_switch{
+	pixel_x = 11;
+	pixel_y = 22
+	},
+/obj/structure/cable{
+	icon_state = "0-8"
 	},
 /turf/open/floor/plating,
-/area/ship/crew)
+/area/ship/maintenance/starboard)
+"YV" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/effect/turf_decal/siding/white/end,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage)
 "ZC" = (
 /turf/closed/wall/mineral/titanium/nodiagonal,
 /area/ship/maintenance/port)
-"ZN" = (
-/obj/effect/turf_decal/spline/fancy/opaque/black{
+"ZH" = (
+/obj/effect/turf_decal/trimline/opaque/red/filled/warning{
 	dir = 1
 	},
-/turf/open/floor/plasteel/tech,
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/obj/machinery/atmospherics/pipe/manifold4w/supply/hidden/layer2,
+/obj/machinery/atmospherics/pipe/manifold4w/scrubbers/hidden/layer4,
+/turf/open/floor/plasteel/white,
 /area/ship/medical)
-"ZO" = (
-/obj/machinery/vending/cola/random,
+"ZJ" = (
 /obj/effect/turf_decal/trimline/opaque/red/filled/line,
-/obj/machinery/firealarm/directional/east,
-/turf/open/floor/plasteel,
-/area/ship/cargo)
+/obj/item/radio/intercom/wideband/directional/south,
```

Comment:
Why is there a wideband relay in the central hallway? Just stick with having one on the bridge. We have wideband logs and the ship has two officers anyway.

Reply:
Old holdout from the original Li Tieguai, removed.